src/services/inplayer.account.service.ts

```diff
@@ -52,6 +52,34 @@ export const getUser = async (): Promise<Customer> => {
 
 export const getFreshJwtToken = async ({ auth }: { auth: AuthData }) => auth;
 
+export const updateCustomer: UpdateCustomer = async (values) => {
+  try {
+    const consents: { [key: string]: string } = {};
+    values.consents?.map((consent: CustomerConsent) => {
+      if (consent.label) {
+        const { customerId, date, newestVersion, needsUpdate, ...rest } = consent;
+        consents[`consents_${consent.name}`] = JSON.stringify(rest);
+      }
+    });
+    const response = await InPlayer.Account.updateAccount({
+      fullName: `${values.firstName} ${values.lastName}`,
```

Comment:
I see that fullname is a built in property in the inplayer api. I'm a little worried about denormalizing the data in to 2 sets of fields that it can get out of sync.  Especially if anyone is using inplayer api's as part of a partner integration or another app besides this web app.

Maybe we can just use the fullname field and add our own unique delimiter like a tab between first and last name or something?  We could store it as a json blob, but that also seems problematic. How do you normally work with Fullname? Have you run into this problem before?

Reply:
In this case, we set `fullName` because this is a mandatory field, but it is never used. We set first and last names as a part of the metadata. 
Yes, you are right, it will be problematic for the existing users without first and last names.
For this case, we can have a fallback, if the first/last name is not set, we can use `fullName` with a delimiter( e.g. the first white space )

Reply:
Again though, this api will likely be connected to more apps than just the web app, and I don't want to have to make sure all of them know to use the first/last meta fields and ignore full name. I think it's more maintainable to always use fullname and not add the other fields.

Reply:
this is also fixed I think